netbox_librenms_plugin/views/base/interfaces_view.py

```
@@ -197,6 +332,15 @@ def get_context_data(self, request, obj, interface_name_field, server_key=None):
             ports_data = cached_data.get("ports", [])
             matched_interface_ids = set()
 
+            # Build port_stack relationship maps from cached data
+            port_stack_relationships = cached_data.get("port_stack_relationships", {})
+            lag_members = port_stack_relationships.get("lag_members", {})
+            sub_interfaces = port_stack_relationships.get("sub_interfaces", {})
+            by_port_id = {p["port_id"]: p for p in ports_data if p.get("port_id")}
+
+            # For device interfaces (not VMs), also select lag and parent FKs
+            _extra_related = [] if self.get_select_related_field(obj) == "virtual_machine" else ["lag", "parent"]
```

Flake8 (7.3.0)

[error] local variable '_extra_related' is assigned to but never used

(F841)